api/src/convert/amount.rs

```diff
@@ -51,13 +52,17 @@ impl TryFrom<&external::MaskedAmount> for MaskedAmountV1 {
     type Error = ConversionError;
 
     fn try_from(source: &external::MaskedAmount) -> Result<Self, Self::Error> {
-        let commitment = CompressedCommitment::try_from(source.get_commitment())?;
-        let masked_value = source.get_masked_value();
-        let masked_token_id = source.get_masked_token_id();
+        let commitment = source
+            .commitment
+            .as_ref()
+            .unwrap_or(&Default::default())
+            .try_into()?;
```

Comment:
the generated protobuf implementation of `get_commitment()` and other `get_x()` methods
They have a dedicated default instance they use in the `or_else()` clause, this just uses the more generic `Default::default()` generating it in place to use the `or_else()` would require manually creating the instance prior in order to pass a ref to it so it's about the same
```rust
    pub fn get_commitment(&self) -> &CompressedRistretto {
        self.commitment.as_ref().unwrap_or_else(|| <CompressedRistretto as ::protobuf::Message>::default_instance())
    }
    
        fn default_instance() -> &'static CompressedRistretto {
        static instance: ::protobuf::rt::LazyV2<CompressedRistretto> = ::protobuf::rt::LazyV2::INIT;
        instance.get(CompressedRistretto::new)
    }

```